src/views/mobile.ts

```diff
@@ -1,12 +1,45 @@
 import { IObservable, IObserver } from '../utils/observable/types';
 import { IView } from './types';
+import { NewsState } from '../state/news';
+import { WeatherState } from '../state/weather';
 
 export class MobileView implements IObserver, IView {
+    private dates: Set<IObservable> = new Set();
+    private lastContent: string | undefined;
```

Comment:
Можно переписать в таком виде
```suggestion
    private lastContent?: string;
```

Reply:
А почему не стал так делать?